app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -551,16 +551,22 @@ class NewMessageViewModel @Inject constructor(
             return getStringArrayExtra(recipientsFlag)?.map { Recipient().initLocalValues(it, it) }
         }
 
+        /**
+         * Each customer app is free to do what he wants, so we sometimes receive mailto fields that are empty,
+         * so we don't want to ignore them, so we return null.
+         */
```

Comment:
```suggestion
        /**
         * Each customer app is free to do what it wants, so we sometimes receive empty `mailTo` fields.
         * Instead of ignoring them, we return `null`.
         */
```